feedback/edxapp_wrapper/backends/courseware_o_v1.py

```diff
@@ -0,0 +1,29 @@
+"""
+Courseware definitions for Open edX Olive release.
+"""
```

Comment:
We use this for our plugins/xblocks because we support multiple releases. That's not necessarily true for other organizations, especially for Open edX repos. Can we try another approach?